src/Api/Billing/Public/Controllers/OrganizationSubscriptionsController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Net;
+using Bit.Api.Models.Public.Response;
+using Bit.Api.Models.Request.Organizations;
+using Bit.Core.Context;
+using Bit.Core.Exceptions;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptions.Interface;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrganizationSubscriptionUpdateRequestModel = Bit.Api.Billing.Public.Models.OrganizationSubscriptionUpdateRequestModel;
+
+namespace Bit.Api.Billing.Public.Controllers;
+
+[Route("public/organizations")]
+[Authorize("Organization")]
+public class OrganizationSubscriptionsController : Controller
```

Comment:
Since this route is going to be `/public/organizations`, I think we'll want to use `OrganizationsController` as the name of the class and the file.